lib/open_project/translations/models/locales_updater.rb

```diff
@@ -80,7 +80,7 @@ def commit_and_push_plugin_repo(debug)
   def upload_english(plugin_name)
     # either add or update the english (js) translation file
     titles = {
-      ENGLISH_TRANSLATION_FILE => plugin_name +' Wording',
+      ENGLISH_TRANSLATION_FILE => plugin_name + ' Wording',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.